app/lib/konnectors.js

```diff
@@ -0,0 +1,61 @@
+/* konnector lib ready to be added to cozy-client-js */
+const KONNECTORS_DOCTYPE = 'io.cozy.konnectors'
+
+const STATE_READY = 'ready'
+
+export function addAccount (cozy, konnector, account) {
+  return Promise.resolve(konnector)
+}
+
+export function fetchManifest (cozy, source) {
+  return cozy.fetchJSON('GET', `/konnectors/manifests?Source=${encodeURIComponent(source)}`)
```

Comment:
My idea is to mimic the signature of cozy-client-js modules. The lib as they are now are directly usable as cozy-client-js module. One of the core mechanism is that all method have the cozy-client instance as first parameter, to ensure function's purity.

The counterpart is that we have to pass `cozy.client` as first parameter as every call, but I found that it was a good idea as we will be able to move those modules to cozy-client-js at almost no cost.